Arashayo/TabletopHaven
# Search bar dropdown

## Related Issue
https://app.clickup.com/t/86bx50r0g

## Changes list
- Implemented this insane Search bar

## Testing instructions

1. Run `node backend/src/createTables.js` 
2. Run `node backend/src/fillGames.js` (wait for at least 200 records to be added for full testing)
3. Run both frontend and backend
4. Go to localhost:3000
5. Type in something in search bar
6. Check out the dropdown search results
7. Check the behavior of the search bar, if it closes properly, if it erases search bar input when redirected to a game
8. Feel free to check the dropdown on smaller screen sizes, should work perfectly fine

## Screenshots (if appropriate):
